src/app/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import styles from './Tabs.module.css';
+
+function getActiveClass(pathname: string) {
+  const location = useLocation();
+  if (!location.pathname.includes(pathname)) {
```

Comment:
This will lead to issues with `/` as pathname.
What about 
```suggestion
  if (location.pathname === pathname) {
```